exporter/osg/osgbake.py

```diff
@@ -47,6 +48,43 @@ def collectValues():
             for block in morph_info:
                 morph_info[block].append(block.value)
 
+    def evaluateActiveShapeKeys(time, times):
+        # Get effective shapes according to current time
+        # and evaluate the factor for each
+        value = 0.0
+        if time > times[-1]: # after the last key_block time
+            previous_block = shape.key_blocks[-1]
+            next_block = None
+        elif time < times[0]: # before the first key_block time
+            previous_block = None
+            next_block = shape.key_blocks[0]
+            value = 1.0
+        else:
+            # general case with index in [0:16] and blending two key_blocks
+            index = bisect.bisect(times, time) - 1
```

Comment:
why not use `bisect.bisect_left`? instead of decrementing the index (also it makes it more explicit what's the bisection behaviro you actually use)